migrate gitter links to discord

ckipp01 opened the pull request:
This follows up the announcement made on https://www.scala-lang.org/blog/2021/12/21/discord.html
that Discord is now the official Scala chat.

smarter commented:
There's also the "join chat" gitter button on the front page: https://github.com/lampepfl/dotty/blob/master/docs/_layouts/main.html (and I see references to gitter in css files that could be deleted with that too)

ckipp01 commented:
> There's also the "join chat" gitter button on the front page: https://github.com/lampepfl/dotty/blob/master/docs/_layouts/main.html (and I see references to gitter in css files that could be deleted with that too)

Good catch, I removed a bunch more stuff although I have no idea where that gitter pop out happens. Clicking around and looking at the current site I have no idea where that happens. Am I missing it somewhere? For now I just removed all the gitter stuff.

smarter commented:
On http://dotty.epfl.ch/docs/ on the right there is a "OPEN CHAT" button vertically.

ckipp01 commented:
> On http://dotty.epfl.ch/docs/ on the right there is a "OPEN CHAT" button vertically.

Ahh pff of course I glazed right over it. 👍🏼 thanks

smarter commented:
Hmm the discord logo is huge: http://dotty.epfl.ch/docs/

ckipp01 commented:
> Hmm the discord logo is huge: http://dotty.epfl.ch/docs/

Really? What are you seeing? I'm seeing:

#### Firefox
<img width="929" alt="Screenshot 2022-01-05 at 18 37 11" src="https://user-images.githubusercontent.com/13974112/148263050-c1eb0731-86d0-4df5-863b-b3ce60b692a7.png">

#### Chrome
<img width="1020" alt="Screenshot 2022-01-05 at 18 35 01" src="https://user-images.githubusercontent.com/13974112/148262914-13fd682d-f606-4218-a489-04e83c7c2276.png">

#### Mobile

<img width="370" alt="Screenshot 2022-01-05 at 18 35 51" src="https://user-images.githubusercontent.com/13974112/148263110-512f247c-9e10-4548-8aa0-afd4bdf521d0.png">

SethTisue commented:
perhaps it simply appears large to Guillaume because he finds it threatening ;-)

smarter commented:
In firefox (chrome looks fine):
![Screenshot_20220105_184339](https://user-images.githubusercontent.com/63430/148263882-490b0224-dc4a-4694-95e5-7cfc774aabc4.png)
I use 2x scaling (that is, `window.devicePixelRatio` returns `2`), don't know if that's related.

ckipp01 commented:
I have no idea. I've been playing around trying to mimic this on firefox and I cant.
<img width="1158" alt="Screenshot 2022-01-05 at 20 17 30" src="https://user-images.githubusercontent.com/13974112/148275810-97a94146-068b-44d7-9e17-b061f3b84f8e.png">

What version of Firefox are you using?

smarter commented:
I'm using firefox developer edition (the beta branch) at version 96.0b10. If we can't figure out we could just resize the social icons to all be the same 32x32 size?

ckipp01 commented:
> I'm using firefox developer edition (the beta branch) at version 96.0b10. If we can't figure out we could just resize the social icons to all be the same 32x32 size?

They are already set in the css as 
```css
.social-icon img {
  height: 20px;
  width: 20px;
}
```

smarter commented:
Oh, I figured it out, I use an adblocker and it has a rule that ends up blocking social-links.css :sweat:. So we could rename that file to not contain the word "social" or anything else in https://secure.fanboy.co.nz/fanboy-annoyance.txt
